app/modules/site_settings/models.py

```diff
@@ -106,17 +176,18 @@ def set_key_value(cls, key, value):
 
         assert key in cls.HOUSTON_SETTINGS.keys()
         if not isinstance(value, cls.HOUSTON_SETTINGS[key]['type']):
-            msg = f'Houston Setting key={key}, value incorrect type value={value}'
+            msg = f'Houston Setting key={key}, value incorrect type value={value},'
+            msg += f'needs to be {cls.HOUSTON_SETTINGS[key]["type"]}'
             raise HoustonException(log, msg)
 
         if isinstance(value, str):
             log.debug(f'updating Houston Setting key={key}')
             cls.set(key, string=value, public=cls.HOUSTON_SETTINGS[key]['public'])
-        elif isinstance(value, dict):
+        elif isinstance(value, dict) or isinstance(value, list):
```

Comment:
@naknomum I really want your opinion on this change. What was there just looked wrong. Please comment